src/yashCJun20/Assignment20/CheckArmstrongNum.java

```diff
@@ -0,0 +1,26 @@
+package yashCJun20.Assignment20;
+
+public class CheckArmstrongNum {
+
+	public static void main(String[] args) {
+		
+		int number=153;
+		int reminder=0;
+		int sum=0;
+		int num=0;
+		num=number;
+		while (number!=0) {
+		
+			reminder=number % 10;
+			number=number/10;
+			sum=sum+(reminder*reminder*reminder);
+			
```

Comment:
void checkNumberIsArmstrongOrNot(int number) {
		double sum =0; 
		int num= number;
		while(num>0) {
			sum = Math.pow(num%10, 3) + sum; 
			num = num/10;
		}
		if(number==sum)
			System.out.println("The given number "+number+" is a Armastrong number.");
		else
			System.out.println("The given number "+number+" is not a Armastrong number.");
	}
please compare this code. And made changes accordingly.